test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -194,10 +198,22 @@ private static void EnsureTestData(Options opts, string workingFolder)
             Exec("git", $"clean -xdff", cwd: workingFolder);
 
             var submoduleUpdateFlags = s_isPullRequest ? "" : "--remote";
-            Exec("git", $"{s_gitCmdAuth} submodule set-branch -b {opts.Branch} {s_repositoryName}", cwd: workingFolder, secrets: s_gitCmdAuth);
+            Exec("git", $"{s_gitCmdAuth} submodule set-branch -b {remoteBranch} {s_repositoryName}", cwd: workingFolder, secrets: s_gitCmdAuth);
```

Comment:
For the first-time run of regression test, `submodule set-branch` will crash.
This was found during the 1ES test.